tests/e2e/etcd_config_test.go

```diff
@@ -274,6 +276,117 @@ func TestEtcdPeerNameAuth(t *testing.T) {
 	}
 }
 
+// TestEtcdPeerLocalAddr checks that the inter peer auth works with when
+// the member LocalAddr is set.
+func TestEtcdPeerLocalAddr(t *testing.T) {
+	e2e.SkipInShortMode(t)
+
+	nodeIP, err := netutil.GetDefaultHost()
+	fmt.Println("Got node IP:", nodeIP)
+	if err != nil {
+		t.Fatal("Could not get IP address of system")
+		return
+	}
```

Comment:
This is a generic comment. Please try to simplify the error checking.
```suggestion
	require.NoError(t, err)
```